### PR TITLE
n26265: add a few more questions

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

File: _posts/2022-10-26-#26265.md [hunk 1]
```diff
@@ -57,8 +59,16 @@ discussed this topic.
 1. Between disallowing sizes less than 65 bytes and sizes equal to 64 bytes, which approach do you
    think is better and why? What are the different implications of both approaches?
 
-_More questions to come soon!_
+1. What is "OP_RETURN padding"? How is `MIN_PADDING` calculated
+   [here](https://github.com/bitcoin-core-review-club/bitcoin/commit/e5adc1a284a9292362e75501adc9f71ac6ecdf6e#diff-28d3b20588dcdd115f47a2c83254a9dcf413b6b6dbe5ad1833e922a042b0023eR34),
+i.e., what items are 10, 9, and 41 bytes?
```
Comment (stickies-v) [inline _posts/2022-10-26-#26265.md:64]:
```suggestion
1. What is "OP_RETURN padding"? What do 10, 9, and 41 represent in the calculation of [`MIN_PADDING`](https://github.com/bitcoin-core-review-club/bitcoin/commit/e5adc1a284a9292362e75501adc9f71ac6ecdf6e#diff-28d3b20588dcdd115f47a2c83254a9dcf413b6b6dbe5ad1833e922a042b0023eR34)
```

File: _posts/2022-10-26-#26265.md [hunk 2]
```diff
@@ -47,7 +48,8 @@ discussed this topic.
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or
    NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
 
-1. Why was the minimum transaction size 85 bytes?
+1. Why was the minimum transaction size 85 bytes? Can you describe the attack? Why does setting this
+   policy help prevent the attack? Does it eliminate the possibility?
```
Comment (stickies-v) [inline _posts/2022-10-26-#26265.md:52]:
```suggestion
   policy help prevent the attack? Does it eliminate the attack vector entirely?
```

**Review by stickies-v — APPROVED** [review]
LGTM, just a few phrasing suggestions

**Comment by glozow** [comment]
Thanks @stickies-v, took the suggestions

**Pull request merged**